src/get-project-options.js

```diff
@@ -34,6 +34,22 @@ function getProjectType(checkForDep, keywords) {
   throw new Error('Ember CLI project type could not be determined');
 }
 
+/**
+ * Check if there is a yarn.lock file to indicate if the project uses yarn as the package manager
+ *
+ * @param {string} projectRoot - Path to the project root to check for yarn usage
+ * @returns {Promise<boolean>}
+ */
+module.exports.hasYarn = async function hasYarn(projectRoot) {
```

Comment:
I'm tempted to move has-yarn to its own file so that none of the getProjectOptions imports have to change.

Reply:
I'll defer to your judgment, but I personally like keeping related things in the same file for code readability but that could be a personal preference thing. Let me know if you want it separated.

Reply:
yea let's do it :)